src/main/scala/com/webtrends/harness/component/zookeeper/discoverable/WeightedCommand.scala

```diff
@@ -3,19 +3,21 @@ package com.webtrends.harness.component.zookeeper.discoverable
 import com.webtrends.harness.command.Command
 
 import scala.concurrent.Future
+import scala.reflect.ClassTag
 
 case class Weight(weight: Int, forceSet: Boolean)
 
-trait WeightedCommand extends Command {
+abstract class WeightedCommand[Input <: Product : ClassTag, Output <: Any : ClassTag]
+  extends Command[Input, Output] {
```

Comment:
Is this needed anymore? We were supposed to be doing away with any commands outside of the wookiee-core internal command.